network_observability/observing-network-traffic.adoc

```diff
@@ -16,19 +16,17 @@ include::modules/network-observability-pktdrop-overview.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../network_observability/observing-network-traffic.adoc#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops]
+* For more information about configuring packet drops in the `FlowCollector`, see xref:../network_observability/observing-network-traffic.adoc#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
+* For more information about viewing packet drop metrics in *Observe* -> *Dashboards*, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Viewing Network Observability metrics dashboards]
+[Network Observability metrics]
```

Comment:
It looks like this was meant to be replaced in the link text above instead of floating here